addon/templates/components/frost-guage.hbs

```diff
@@ -0,0 +1,36 @@
+<svg height='{{size}}' width='{{size}}' viewPort='0 0 {{size}} {{size}}' version='1.1' xmlns='http://www.w3.org/2000/svg'>
+  <circle fill='transparent' class='track' cx={{center}} cy={{center}} r={{radius}} stroke-dasharray='{{circumference}}'></circle>
+  {{#if arcs}}
+    {{#each arcs as |arc|}}
+      <circle transform='{{arc.transform}}' stroke='{{arc.color}}' fill='transparent' class='fill {{arc.classes}}' cx={{center}} cy={{center}} r={{radius}} stroke-dasharray='{{circumference}}' stroke-dashoffset='{{arc.offset}}'></circle>
+    {{/each}}
+  {{/if}}
+
+  {{#if scale}}
+    {{#each ticks as |tick|}}
+      <line
+        x1={{tick.x1}}
+        x2={{tick.x2}}
+        y1={{tick.y1}}
+        y2={{tick.y2}}
+        stroke="white"
+        fill="transparent"
+        transform="rotate({{tick.rotation}} {{tick.orient}} {{tick.orient}})"
+        stroke-width="1"
+      />
+    {{/each}}
+  {{/if}}
+</svg>
+{{#if scale}}
```

Comment:
Can you show us the difference between if scale or not (screen capture)? I'm confused.  Why are we using `{{style}}` and in the binding of the component `style` attribute. Is that necessary?